R/metadata.R

```diff
@@ -250,27 +271,36 @@ setClass(
 #' @family metadata
 metadata <- function(constituent, flow, load.rate="", dates, 
                      conc.units, flow.units, load.units, load.rate.units, 
-                     station="", custom=NULL, validate=TRUE) {
+                     station="", custom=NULL, validate=TRUE, consti.name="",
+                     wq.sta.id="", station.lat=NA, station.lon=NA,
+                     disch.sta.id="", disch.basin.area=NA, wq.basin.area=NA) {
```

Comment:
let's make the default for `disch.sta.id` be `wq.sta.id`, and the default for `disch.basin.area` be `wq.basin.area`. This will require swapping `disch.basin.area` and `wq.basin.area` in the argument list.